srcpkgs/freedoom/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'freedoom'
+pkgname=freedoom
+version=0.12.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="asciidoc deutex python3 python3-Pillow"
+short_desc="Free game based on the Doom engine"
+maintainer="Michael Taboada <voidpkgs@forwardme.email>"
+license="BSD-3-Clause"
+homepage="https://freedoom.github.io/"
+distfiles="https://github.com/$pkgname/$pkgname/releases/download/v$version/$pkgname-$version.tar.xz"
+checksum=bad733695664c699fc7f02c833916c889b080fadc3b503c40f0aaba3bf637380
+
+pre_build() {
+	# CPP doesn't seem to be used for a c preprocessor in this makefile (?), so replacing the value in the makefile breaks the build.
+	make_build_args="CPP=${XBPS_BUILDDIR}/freedoom-${version}/scripts/simplecpp"
+}
```

Comment:
```suggestion
make_build_args="CPP='scripts/simplecpp'"
```
and put it after build_style

Reply:
Using scripts/simplecpp breaks when make chdirs to another directory, no idea how the makefile normally works at all lol.